cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/sponge/SpongeCloudNetBridgePlugin.java

```diff
@@ -27,29 +27,6 @@
 
     @Listener
     public synchronized void handle(GameStartedServerEvent event) {
-        try {
-            Method method = ClassLoader.getSystemClassLoader().getClass().getMethod("getCloudNetWrapperClassLoader");
-            method.setAccessible(true);
-
-            ClassLoader classLoader = new SpongeCloudNetAdapterClassLoader(
-                    (ClassLoader) method.invoke(ClassLoader.getSystemClassLoader()),
-                    Thread.currentThread().getContextClassLoader(),
-                    ClassLoader.getSystemClassLoader().getParent()
-            );
-
-            Field field = ClassLoader.class.getDeclaredField("parent");
-            field.setAccessible(true);
-            field.set(Thread.currentThread().getContextClassLoader(), classLoader);
-
-
-            field = ClassLoader.class.getDeclaredField("scl");
-            field.setAccessible(true);
-            field.set(null, Thread.currentThread().getContextClassLoader());
-
-        } catch (Exception ex) {
-            ex.printStackTrace();
-        }
-
```

Comment:
I dont think that this whole code was just unnecessary, removing it might cause problems with sponge.

Reply:
Well by adding the jar file to class path it make sence that this whole stuff won't be needed anymore. But i'll test it hardly this evening on some test sessions, wait for reply